esy.json

```diff
@@ -18,13 +18,22 @@
   "devDependencies": {
     "@opam/merlin": "^3.2.2",
     "@opam/merlin-lsp": "*",
-    "ocaml": "~4.7.1000"
+    "ocaml": "~4.7.1000",
+    "@opam/httpkit": "*",
+    "@opam/httpkit-lwt": "*",
+    "@opam/logs": "*",
+    "@opam/fmt": "*",
+    "@opam/yojson": "*"
   },
   "resolutions": {
     "@brisk/core": "link-dev:./core",
     "@brisk/macos": "link-dev:./renderer-macos",
     "@opam/dune": "ocaml/dune:dune.opam#01b46a3",
     "@opam/merlin-lsp": "ocaml/merlin:merlin-lsp.opam#3e34bb5",
-    "brisk-reconciler": "briskml/brisk-reconciler#fa605f1"
+    "brisk-reconciler": "briskml/brisk-reconciler#fa605f1",
+    "@opam/httpkit": "ostera/httpkit:httpkit.opam#322ca26",
```

Comment:
Whatever you add to the `resolutions`, try to keep order alphabetical (scopes go first).

Reply:
I'm opposed to this rule. Any rule you cannot automate is just mental overhead!

Reply:
I'd prefer to sort them by significance but that's what would be a mental overhead, so alphabetical seems like a good compromise. (most package managers do it automatically, maybe one day we'll get that for esy)

It's not like we'll add dependencies on every single PR.

Reply:
I'm still opposed to it - I'll never remember/care about such artificial rule.

Reply:
Aren't there enough artificial rules in this world 😄 ?

Reply:
It's a convention, easy enough to follow.

Reply:
![nerds](https://media.giphy.com/media/c4t11obaChpu0/giphy.gif)

Reply:
Ok, let's postpone the manual enforcing. We'll come up with a way to automate it.

Reply:
Don't fight guys :D Ok, what's about to keep the  `opam` packages at the top and the rest of packages alphabetical?

Reply:
That’s what I’d usually do, but then there are scoped npm packages that mess up this logic.